release/test_releases_json.sh

```diff
@@ -8,14 +8,20 @@ function main {
 	set_up
 
 	test_merge_json_snippets dxp
+	test_process_new_product
 	test_promote_product_versions dxp
 
 	tear_down
 }
 
 function set_up {
+	export LIFERAY_RELEASE_PRODUCT_NAME="dxp"
+	export _PRODUCT_VERSION="2024.q1.13"
```

Comment:
@kiwm, can you please use a very future release, like `2024.q1.9999`, to prevent issues with this test?